wikipedia-dark.user.css

```diff
@@ -5849,7 +5861,13 @@ regexp("^https://(foundation|donate)\.wikimedia\.org.*$") {
   }
   figure[typeof~="mw:Image/Thumb"], figure[typeof~="mw:Video/Thumb"],
   figure[typeof~="mw:Audio/Thumb"], figure[typeof~="mw:Image/Frame"],
-  figure[typeof~="mw:Video/Frame"], figure[typeof~="mw:Audio/Frame"] {
+  figure[typeof~="mw:Video/Frame"], figure[typeof~="mw:Audio/Frame"],
+  figure[typeof~="mw:Image/Thumb"] > figcaption,
+  figure[typeof~="mw:Video/Thumb"] > figcaption,
+  figure[typeof~="mw:Audio/Thumb"] > figcaption,
+  figure[typeof~="mw:Image/Frame"] > figcaption,
+  figure[typeof~="mw:Video/Frame"] > figcaption,
+  figure[typeof*="mw"] > figcaption {
```

Comment:
figure[typeof*="mw"] > figcaption

replaces 

  figure[typeof~="mw:Image/Thumb"], figure[typeof~="mw:Video/Thumb"],
  figure[typeof~="mw:Audio/Thumb"], figure[typeof~="mw:Image/Frame"],
  figure[typeof~="mw:Video/Frame"], figure[typeof~="mw:Audio/Frame"] {
  figure[typeof~="mw:Video/Frame"], figure[typeof~="mw:Audio/Frame"],
  figure[typeof~="mw:Image/Thumb"] > figcaption,
  figure[typeof~="mw:Video/Thumb"] > figcaption,
  figure[typeof~="mw:Audio/Thumb"] > figcaption,
  figure[typeof~="mw:Image/Frame"] > figcaption,
  figure[typeof~="mw:Video/Frame"] > figcaption,

fyi

Reply:
I was not sure because I made that fix quite long ago, but if I remember well there were few "figure types" that shouldn't be themed due to an error. Unfortunately I can't find now any article with them.

I think in current form it is not a a big excessing code.

Reply:
If it turns out later that we should be applying this fix to all figures, we can replace the specific selectors with more generic ones later. The `figcaption` selectors could be consolidated though, as per above.

Looks fine to me otherwise.